src/index.ts

```diff
@@ -160,5 +165,11 @@ require('dotenv').config()
     filterFile
   })
 
-  portman.run()
+  if (options.uploadOnly) {
```

Comment:
not sure if this is the preferred way to split the different code paths. newmanOnly would be another method would be put here. Would a switch case be preferred?

Reply:
this is fine for now. a case statement would be messy as you're checking for diff options instead of one